### DISCUSSION TITLE
Friends I have problems with module java.base does not "opens jdk.internal.misc" to unnamed module

### DISCUSSION THREAD

**Discussion by jolucode** [quarkusio/quarkus#42576] [Community]
Unable to make field private static final jdk.internal.misc.Unsafe jdk.internal.misc.Unsafe.theUnsafe accessible: module java.base does not "opens jdk.internal.misc" to unnamed module @20bd8be5

I traid to config VM:

![image](https://github.com/user-attachments/assets/663dba36-4b24-4eb8-9182-476256ceb765)

but the problems continuos please Anything help can me.

**Comment by sberyozkin** [comment]
CC @zakkak @galderz

**Comment by galderz** [comment]
@sberyozkin Not sure why you cc'd us but not really to do with native.

@jolucode You need to figure out why you need access to Unsafe and talk to the corresponding developers to understand if you really need and how to get access to it. We're not aware of anything within Quarkus that would require you to enable unsafe access explicitly.

**Reply by sberyozkin** [reply]
@galderz Sorry, my initial impression this problem was showing itself in native, but I got it wrong

**Comment by starksm64** [comment]
What is the full stacktrace showing where this error message is coming from?
